go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/grafana/grafana-plugin-sdk-go
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.3
 
 require (
-	github.com/apache/arrow/go/v15 v15.0.2
+	github.com/apache/arrow-go/v18 v18.0.0
```

Comment:
can we use the latest commit on main of arrow-go so it includes [this](https://github.com/apache/arrow-go/pull/199)?

Reply:
@ryantxu related PR https://github.com/grafana/grafana-plugin-sdk-go/pull/1165